addons/mail/models/mail_thread.py

```diff
@@ -2629,37 +2630,51 @@ def _notify_classify_recipients(self, recipient_data, model_name, msg_vals=None)
         }]
         only return groups with recipients
         """
+        recipient_lang_groups = self.env['res.partner'].sudo().read_group(
```

Comment:
Side note: "groups" suffix is misleading, as we are already dealing with "groups" (using notify meaning of groups).